week-1/exercises/decimal.js

```diff
@@ -8,3 +7,11 @@ const num = 56.5467;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+const wholeNumberPart = Math.floor(num); // Extract the whole number part 56
+const decimalPart = num - wholeNumberPart; // Calculate the decimal part; num is 56.5467 - wholeNum is 56 so 0,5467 is decimal
```

Comment:
Your thinking here is correct. However, there are some nuances with using floating point numbers, that you will see when this code is run.
To get the right answer, try looking up how to maintain number of decimal places when subtracting floating point numbers in javascript